src/planning.ts

```diff
@@ -336,7 +336,7 @@ export class Plan {
         // TODO: Remove this hack by storing the pen-up/pen-down heights
         // in a single place, and reference them from the PenMotions.
         if (j === this.motions.length - 1) {
-          return new PenMotion(this.minPenPosition, penUpHeight, motion.duration())
+          return new PenMotion(Axidraw.penPctToPos(0), penUpHeight, motion.duration())
```

Comment:
We don't want to hard-code this, but I wanted to make it explicit where the logic change had occurred.

Reply:
oh! good catch, yeah this should be fixed. Honestly I have not looked into why PenMotions need a starting position in their motion definition - we should be able to know what the current position is since we only ever configure 2 pen positions - up and down - and use that to calculate the target duration delay.